.travis.yml

```diff
@@ -1,5 +1,6 @@
 language: go
 
 script:
+- make bootstrap
```

Comment:
It looks like there's a way to inject go lint here. 
https://gist.github.com/y0ssar1an/df2dab474520c4086926f672c52db139

Reply:
Also since this PR requires golang 1.10+, there's a way to do that too apparently :)